jsonld.js/src/test/scala/org/w3/banana/jsonldjs/io/JsonLdJsParserTest.scala

```diff
@@ -28,10 +27,9 @@ object JsonLdJsParserTest extends WordSpec with Matchers {
       import org.w3.banana.binder.ToURI.URIToURI
       val graph = (
         BNode()
-          -- schema("name") ->- "Manu Sporny"
-          -- schema("url") ->- URI("http://manu.sporny.org/")
-          -- schema("image") ->- URI("http://manu.sporny.org/images/manu.png")
-      ).graph
+        -- schema("name") ->- "Manu Sporny"
+        -- schema("url") ->- URI("http://manu.sporny.org/")
+        -- schema("image") ->- URI("http://manu.sporny.org/images/manu.png")).graph
 
```

Comment:
I am not sure the above is an improvement...

Reply:
this is the result of scalariform run, we can omit that (maybe we have to tune default settings?)